src/commands/trigger-workflows-for-all-modified.yml

```diff
@@ -289,7 +290,10 @@ steps:
             running=$(jq -s 'map(select(.status == "running")) | length' ${workflow_status})
             success=$(jq -s 'map(select(.status == "success")) | length' ${workflow_status})
             echo "total: ${total}, running: ${running}, success: ${success}"
-            if (( $total == 0 || $running > 0 )); then
+            if (( $total == 0 && $running == 0 )); then
+              echo "No running workflows, return success"
+              exit 0
+            elif (( $total == 0 || $running > 0 )); then
```

Comment:
Given the change above, is there still a good reason to sleep if total == 0 ?

Reply:
probably needs to be `total ==0 && running >0`

Reply:
Fixed